src/components/ui-shell/_navigation-menu.scss

```diff
@@ -104,12 +109,12 @@
   }
 
   .#{$prefix}--navigation__category-toggle:hover {
-    background-color: $ibm-colors__gray--80;
+    background-color: #333333;
```

Comment:
yeah I'm not sure if we have an existing token for this. I searched the codebase for this value and it seems to be hard coded everywhere so far. I think it is definitely worth tokenizing this value though

Reply:
Yes, I believe its one of the interactive colors. Or at least it was when I made the shell, I think they've must have updated that palette because I can't find it anymore the closest is gray 90 hover @ `#353535`

Reply:
should I update all references to #333333 to gray 90 hover instead then?